content/en/blog/_posts/2018-07-12-resizing-persistent-volumes-using-kubernetes.md

```diff
@@ -0,0 +1,82 @@
+
+```
+~> cat standard.yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: standard
+parameters:
+  type: pd-standard
+provisioner: kubernetes.io/gce-pd
+allowVolumeExpansion: true
+reclaimPolicy: Delete
+```
+
+Any PVC created from this StorageClass can be edited (as illustrated below) to request more space. Kubernetes will interpret a change to the storage field as a request for more space, and will trigger automatic volume resizing.
+
+![PVC StorageClass](/images/blog/2018-07-12-resizing-persistent-volumes-using-kubernetes/pvc-storageclass.png)
+
+## File System Expansion
+
+Block storage volume types such as GCE-PD, AWS-EBS, Azure Disk, Cinder, and Ceph RBD typically require a file system expansion before the additional space of an expanded volume is usable by pods. Kubernetes takes care of this automatically whenever the pod(s) referencing your volume are restarted.
+
+Network attached file systems (like Glusterfs and Azure File) can be expanded without having to restart the referencing Pod, because these systems do not require special file system expansion.
+
+File system expansion must be triggered by terminating the pod using the volume. More specifically:
+
+* Edit the PVC to request more space.
+* Once underlying volume has been expanded by the storage provider, then the PersistentVolume object will reflect the updated size and PVC will have the `FileSystemResizePending` condition.
+
+You can verify this by running `kubectl get pvc <pvc_name> -o yaml`
+
+```
+~> kubectl get pvc myclaim -o yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: myclaim
+  namespace: default
+  uid: 02d4aa83-83cd-11e8-909d-42010af00004
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 14Gi
+  storageClassName: standard
+  volumeName: pvc-xxx
+status:
+  capacity:
+    storage: 9G
+  conditions:
+  - lastProbeTime: null
+    lastTransitionTime: 2018-07-11T14:51:10Z
+    message: Waiting for user to (re-)start a pod to finish file system resize of
+      volume on node.
+    status: "True"
+    type: FileSystemResizePending
+  phase: Bound
+```
+* Once PVC has the condition `FileSystemResizePending` then pod that uses the PVC can be restarted to finish file system resizing on the node. Restart can be achieved  by scaling down the deployment and scaling it up or by deleting and recreating the pod.
```

Comment:
`Once PVC` -> `Once the PVC`

Reply:
there is a double space after `achieved`.

i would modify the last sentence to:
> Restart can be achieved by deleting and recreating the pod or by scaling down the deployment and then scaling it up again.